core/Cargo.toml

```diff
@@ -33,13 +33,13 @@ include = [
 
 [dependencies]
 parquet-format = "4.0.0" # This must be kept in sync with that from parquet crate
-arrow = { version = "52.0.0", features = ["prettyprint", "ffi", "chrono-tz"] }
-arrow-array = { version = "52.0.0" }
-arrow-buffer = { version = "52.0.0" }
-arrow-data = { version = "52.0.0" }
-arrow-schema = { version = "52.0.0" }
-arrow-string = { version = "52.0.0" }
-parquet = { version = "52.0.0", default-features = false, features = ["experimental"] }
+arrow = { git = "https://github.com/tustvold/arrow-rs.git", rev = "78b6139", features = ["prettyprint", "ffi", "chrono-tz"] }
+arrow-array = { git = "https://github.com/tustvold/arrow-rs.git", rev = "78b6139" }
```

Comment:
Now that the PR is merged, can we switch this to a revision of the official arrow-rs repo?

Reply:
Yes. We can point to arrow-rs repo.